<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1221</width>
    <height>644</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLineEdit{
background: transparent;
border: none;
border-bottom: 1px solid rgba(0, 0, 0, 155);
font-family: sans-serif;
}
QMainWindow{
background: rgb(0, 90, 106);
font-family: sans-serif;
}

QGroupBox{
background: white;
border: 1px solid rgba(255, 255, 255, 200);
font-family: sans-serif;
border-radius:5px;
}
QGroupBox#statusBox{
border-radius: 0px;
}
QPushButton#exitButton, QPushButton#minimizeButton{
border-radius: 0px;
border: none;
background: transparent;
}
QPushButton#exitButton:hover{
background: rgb(255, 46, 46);
}
QPushButton#minimizeButton:hover{
background: rgb(223, 223, 223);
}
QPushButton:hover{
background: rgb(75, 195, 220);
border: none;
}

QPushButton{
background: rgb(58, 180, 208);
border-radius: 10px;
border: 1px solid rgb(18, 140, 168);
font-family: sans-serif;
}

QTableView, QListView{
background: transparent;
font-family: sans-serif;
height: 111px;
border-radius:5px;
outline: 0;
spacing: 10px;
}

QAbstractItemView{
background:transparent;
outline: 0;

}

QAbstractItemView:item{
background: rgb(0, 81, 95);
border-radius: 5px;
color: white;
padding:5px;
outline: 0;
margin: 2px;

}

QAbstractItemView:item:hover{
background: rgb(0, 110, 127);
}

QAbstractItemView:item:selected{
background: rgb(0, 81, 95);
}
QTableCornerButton::section{
background: transparent;
}
QProgressBar {
    border: 2px solid grey;
    border-radius: 5px;
    text-align: center;
}

QProgressBar::chunk {
    background-color: #05B8CC;
    width: 20px;
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>1160</width>
      <height>560</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QListWidget" name="courcesList">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>256</width>
       <height>511</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="autoScrollMargin">
      <number>15</number>
     </property>
     <property name="dragEnabled">
      <bool>false</bool>
     </property>
     <property name="dragDropOverwriteMode">
      <bool>false</bool>
     </property>
     <property name="alternatingRowColors">
      <bool>false</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::NoSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectColumns</enum>
     </property>
     <property name="isWrapping" stdset="0">
      <bool>false</bool>
     </property>
     <property name="resizeMode">
      <enum>QListView::Adjust</enum>
     </property>
     <property name="spacing">
      <number>2</number>
     </property>
     <property name="sortingEnabled">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QListWidget" name="listWidget_2">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>20</y>
       <width>781</width>
       <height>511</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="autoScrollMargin">
      <number>15</number>
     </property>
     <property name="alternatingRowColors">
      <bool>false</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::NoSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectColumns</enum>
     </property>
     <property name="resizeMode">
      <enum>QListView::Adjust</enum>
     </property>
     <property name="spacing">
      <number>2</number>
     </property>
     <property name="batchSize">
      <number>100</number>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="statusBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1221</width>
      <height>30</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="exitButton">
     <property name="geometry">
      <rect>
       <x>1190</x>
       <y>0</y>
       <width>30</width>
       <height>27</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>sans-serif</family>
      </font>
     </property>
     <property name="text">
      <string>X</string>
     </property>
    </widget>
    <widget class="QPushButton" name="minimizeButton">
     <property name="geometry">
      <rect>
       <x>1160</x>
       <y>0</y>
       <width>30</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>—</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>5</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Sans Serif</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Проект</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
